nominatim/tools/special_phrases/sp_wiki_loader.py

```diff
@@ -46,7 +46,8 @@ def __init__(self, config: Configuration) -> None:
                          ['af', 'ar', 'br', 'ca', 'cs', 'de', 'en', 'es',
                           'et', 'eu', 'fa', 'fi', 'fr', 'gl', 'hr', 'hu',
                           'ia', 'is', 'it', 'ja', 'mk', 'nl', 'no', 'pl',
-                          'ps', 'pt', 'ru', 'sk', 'sl', 'sv', 'uk', 'vi']
+                          'ps', 'pt', 'ru', 'sk', 'sl', 'sv', 'uk', 'vi',
+                          'lv', 'tr', 'de-at']
```

Comment:
Let's not start with with allowing dialects.

Reply:
I removed 'de-at' now.